src/Sylius/Behat/Context/Api/EmailContext.php

```diff
@@ -169,11 +191,24 @@ public function recipientShouldReceiveNoEmails(string $recipient): void
         Assert::false($this->emailChecker->hasRecipient($recipient));
     }
 
+    /**
+     * @Then only one email should have been sent to :recipientEmail
+     */
+    public function onlyOneEmailShouldHaveBeenSentTo(string $recipient): void
```

Comment:
```suggestion
     * @Then only one email should have been sent to :recipient
     */
    public function onlyOneEmailShouldHaveBeenSentTo(string $recipient): void
```
I'd go the opposite way to be in line with the other methods